javascript/javascript1/week3/when.js

```diff
@@ -0,0 +1,13 @@
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+  };
+  function timeToReach(obj){
```

Comment:
Try to name the functions a little more precise and descriptive. Start with a verb always and think about what the function is doing.

Reply:
done